presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -36,6 +36,8 @@ Currently, the following statistics are available in Presto:
    * **data size**: the data size that needs to be read
    * **null count**: the number of null values
    * **distinct value count**: the estimated number of distinct values
+   * **low value**: the smallest value in the column
+   * **high value**: the largest value in the column
```

Comment:
Consider putting "Return low/high value in show stats" commit on hold, so that there remains an open path for showing stats for actual plan in `SHOW STATS FOR `